From 5ba8700b129d6cd87159d10da734cf6108937efb Mon Sep 17 00:00:00 2001
From: Fabien Parent <fparent@baylibre.com>
Date: Wed, 6 Apr 2022 11:37:27 +0200
Subject: [PATCH] WORKAROUND: drm/mediatek: mt8365: disable OVL for DSI path

There is some artifacts when MDP is used on the DSI display
when OVL is enabled. Workaround this issue by disabling
OVL from the DSI path.

Signed-off-by: Fabien Parent <fparent@baylibre.com>
---
 drivers/gpu/drm/mediatek/mtk_drm_drv.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/mediatek/mtk_drm_drv.c b/drivers/gpu/drm/mediatek/mtk_drm_drv.c
index fe8b7a0b446a..b21817365179 100644
--- a/drivers/gpu/drm/mediatek/mtk_drm_drv.c
+++ b/drivers/gpu/drm/mediatek/mtk_drm_drv.c
@@ -166,7 +166,6 @@ static const enum mtk_ddp_comp_id mt8183_mtk_ddp_ext[] = {
 };
 
 static const enum mtk_ddp_comp_id mt8365_mtk_ddp_main[] = {
-	DDP_COMPONENT_OVL0,
 	DDP_COMPONENT_RDMA0,
 	DDP_COMPONENT_COLOR0,
 	DDP_COMPONENT_CCORR,
-- 
2.35.1

